### ISSUE TITLE
NativeItem Alternate Title

### ISSUE THREAD

**Issue by matchaicedtea** [justalemon/LemonUI#34]
![image](https://user-images.githubusercontent.com/43957614/130166828-51ef9a1a-41e5-4030-bcca-da4d6aec5fbe.png)

```
public static NativeItem weapon = new NativeItem("Weapon:");
. . .
string wHash = Function.Call<WeaponHash>(Hash.GET_SELECTED_PED_WEAPON, Game.Player.Character).ToString(); // Get current weapon
weapon.AltTitle = $"{SAM_WG.GetDisplayName(wHash)}"; // Update alt. title
```

**Comment by matchaicedtea** [comment]
Alternate Title returns to correct position after going down to the next option.

**Comment by justalemon** [comment]
This has been already fixed in c1484384f91220c9e3f513921d69b3ff3f58c3c1. Expect the new release.
